recipe/meta.yaml

```diff
@@ -1,60 +1,51 @@
 {% set name = "rsa" %}
-{% set version = "3.4.2" %}
-{% set hash_type = "sha256" %}
-{% set hash_val = "25df4e10c263fb88b5ace923dd84bf9aa7f5019687b5e55382ffcdb8bede9db5" %}
+{% set version = "4.0" %}
 
 package:
-  name: {{ name }}
-  version: {{ version }}
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash_val }}
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1a836406405730121ae9823e19c6e806c62bbad73f890574fff50efa4122c487"
 
 build:
-  noarch: python
+  number: 0
   entry_points:
     - pyrsa-priv2pub = rsa.util:private_to_public
     - pyrsa-keygen = rsa.cli:keygen
     - pyrsa-encrypt = rsa.cli:encrypt
     - pyrsa-decrypt = rsa.cli:decrypt
     - pyrsa-sign = rsa.cli:sign
     - pyrsa-verify = rsa.cli:verify
-    - pyrsa-encrypt-bigfile = rsa.cli:encrypt_bigfile
-    - pyrsa-decrypt-bigfile = rsa.cli:decrypt_bigfile
-
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  noarch: "python"
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
 
 requirements:
   host:
-    - python
     - pip
-    - pyasn1 >=0.1.3
-
-  run:
     - python
+  run:
     - pyasn1 >=0.1.3
+    - python
 
 test:
   imports:
     - rsa
-
   commands:
     - pyrsa-priv2pub --help
     - pyrsa-keygen --help
     - pyrsa-encrypt --help
     - pyrsa-decrypt --help
     - pyrsa-sign --help
     - pyrsa-verify --help
-    - pyrsa-encrypt-bigfile --help
-    - pyrsa-decrypt-bigfile --help
 
 about:
   home: https://stuvel.eu/rsa
   license: Apache 2.0
   summary: 'Pure-Python RSA implementation'
+  license_family: APACHE
+  license_file: LICENSE
```

Comment:
should these go next to license?

Reply:
Makes sense, done. Also added myself as maintainer.